$OpenBSD: patch-toolkit_library_libxul-config_mk,v 1.7 2012/02/20 20:21:43 landry Exp $
Ensure systemwide nss is taken
Ensure local libjs is taken (build failure when spidermonkey is installed,
its libjs_static.a is found first in lib search path)
https://bugzilla.mozilla.org/show_bug.cgi?id=646375
--- toolkit/library/libxul-config.mk.orig	Wed Dec  7 06:28:06 2011
+++ toolkit/library/libxul-config.mk	Sat Dec 10 16:53:04 2011
@@ -321,14 +321,15 @@ EXTRA_DSO_LDOPTS += \
 	$(JPEG_LIBS) \
 	$(PNG_LIBS) \
 	$(QCMS_LIBS) \
-	$(MOZ_JS_LIBS) \
-	$(NSS_LIBS) \
+	../../dist/lib/libjs_static.a \
 	$(MOZ_CAIRO_LIBS) \
 	$(MOZ_HARFBUZZ_LIBS) \
 	$(MOZ_OTS_LIBS) \
 	$(MOZ_APP_EXTRA_LIBS) \
 	$(SQLITE_LIBS) \
 	$(NULL)
+
+OS_LIBS	+= $(NSS_LIBS)
 
 ifdef MOZ_NATIVE_ZLIB
 EXTRA_DSO_LDOPTS += $(ZLIB_LIBS)
